roles/systems/nfs_server/tasks/main.yaml

```diff
@@ -76,9 +79,6 @@
         regexp: '^/srv/nfs/kubernetes/sc/default'
         line: /srv/nfs/kubernetes/sc/default    *(rw,no_root_squash,async,insecure,crossmnt,no_subtree_check)
       when: kubernetes_sc_default_file_rc.stdout != "0"
-  tags:
-    - install
-    - update
```

Comment:
Were these dropped?

Reply:
It's the git diff again not properly representing the changes. If you look at the branch directly the key-order of the task was changed and tags was moved up https://github.com/sassoftware/viya4-iac-k8s/blob/IAC-1066/roles/systems/nfs_server/tasks/main.yaml#L50